pkg/security/ptracer/cws.go

```diff
@@ -218,14 +215,30 @@ func StartCWSPtracer(args []string, envs []string, probeAddr string, opts Opts)
 	}
 
 	var (
-		msgChan        = make(chan *ebpfless.Message, 100000)
+		msgDataChan    = make(chan []byte, 100000)
```

Comment:
Switching from a number of msg to a number of bytes, IMO we may want to increase the size of the chan

Reply:
I don;t think that's the case. The chan is a chan of []byte, so the size applies to the count of slices, not underlysing bytes

Reply:
missread